arviz/tests/base_tests/test_data.py

```diff
@@ -873,3 +873,41 @@ def test_id_conversion_args(self):
         assert isinstance(inference_data, InferenceData)
         assert set(inference_data.posterior.coords["Ivies"].values) == set(IVIES)
         assert inference_data.posterior["theta"].dims == ("chain", "draw", "Ivies")
+
+
+
+class TestDataArrayToDataset:
+    def test_1d_dataset(self):
+        size = 100
+        dataset = convert_to_dataset(xr.DataArray(np.random.randn(1, size), dims=('chain', 'draw')))
```

Comment:
you could set the name of the input dataarray in this test to make sure the name is not overwritten during conversion

Reply:
Yes I forgot to test the name. I'll make suitable changes